lnbits/exceptions.py

```diff
@@ -119,3 +130,43 @@ async def invoice_error_handler(request: Request, exc: InvoiceError):
             status_code=520,
             content={"detail": exc.message, "status": exc.status},
         )
+
+    @app.exception_handler(404)
+    async def error_handler_404(request: Request, exc: HTTPException):
+        logger.error(f"404: {request.url.path} {exc.status_code}: {exc.detail}")
+
+        if not _is_browser_request(request):
+            return JSONResponse(
+                status_code=exc.status_code,
+                content={"detail": exc.detail},
+            )
+
+        path = path_segments(request.url.path)[0]
+        status_code = HTTPStatus.NOT_FOUND
+        message: str = "Page not found."
+
+        if path in settings.lnbits_all_extensions_ids:
+            status_code = HTTPStatus.FORBIDDEN
+            message = f"Extension '{path}' not installed. Ask the admin to install it."
+
+        return template_renderer().TemplateResponse(
+            request,
+            "error.html",
+            {"status_code": int(status_code), "message": message},
+            int(status_code),
+        )
+
```

Comment:
why are we returning forbidden on an 404 exception handler?

Reply:
I'm guessing because the path isn't allowed for the user, something along that logic! @motorina0 ?

that is specifically for the extensions, so the error is 404 but since it's related to an extension.... bla bla bla.... should i remove it and keep it as 404?

Reply:
i think 404 here is also fine